L1Trigger/L1TMuonCPPF/BuildFile.xml

```diff
@@ -0,0 +1,35 @@
+<export>
+  <lib name="1"/>
+</export>
+<flags ADD_SUBDIR="1"/>
+<flags LDFLAGS="-lXMLIO"/>
```

Comment:
do you need to explicitly link against this library?

Reply:
No, this line changed